First pass at removing bower refs

JoeShep opened the pull request:
I've done a global purge of references to Bower, per https://trello.com/c/hcUennj7/137-bower-is-obsolete-now-get-rid-of-it

In most cases, it was a fairly simple update. But as noted in the React exercise, I'm less confident of a one-to-one npm replacement. See https://github.com/nashville-software-school/front-end-milestones /5-modern-frameworks/resources/MF_REACT.md

This process also raised an issue about Firebase: https://github.com/nashville-software-school/front-end-milestones /4-rich-browser-applications/resources/RBA_FIREBASE_CONCEPTS.md uses the realtime SDK. I was under the impression that we were sticking to only RESTful API interactions. Input/clarity appreciated

stevebrownlee commented:
@JoeShep Yes, remove references to the realtime SDK.

https://www.npmjs.com/package/babel-preset-react

JoeShep commented:
Updated the Firebase resources. Removed the real-time resource, moved it to a challenge. Also removed a couple of files that were empty or had a single link in them. Once we have more content for those, we could/should add them back

stevebrownlee commented:
@JoeShep :shipit:
